stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/MessageListViewStyle.kt

```diff
@@ -234,9 +252,18 @@ constructor(
         scrollButtonBottomMargin: Int,
         scrollButtonEndMargin: Int,
         disableScrollWhenShowingDialog: Boolean,
-        optionsOverlayEditReactionsMargin: Int,
-        optionsOverlayUserReactionsMargin: Int,
-        optionsOverlayMessageOptionsMargin: Int,
+        optionsOverlayEditReactionsMarginBottom: Int,
```

Comment:
Also a minor nitpicking but here we go bottom - top - start - end, where as we go top - bottom - start - end otherwise.
Consistency is nice but not _as_ important if someone is using named arguments.